lib/omniauth/strategies/saml.rb

```diff
@@ -186,9 +186,12 @@ def handle_response(raw_response, opts, settings)
         @attributes = response.attributes
         @response_object = response
 
+        # With ruby-saml 1.4.2, that line is never reached with an empty name_id on the response
+        # :nocov:
         if @name_id.nil? || @name_id.empty?
           raise OmniAuth::Strategies::SAML::ValidationError.new("SAML response missing 'name_id'")
         end
+        # :nocov:
```

Comment:
I'm not a fan of these type of comments, that really have nothing to do with the code itself, but are just a way to achieve 100% coverage. See [here](https://dev.to/danlebrero/the-tragedy-of-100-code-coverage) for why trying to always achieve 100% coverage can be a bad thing.

I would remove them.